drivers/enc28j60/enc28j60.c

```diff
@@ -248,10 +248,27 @@ static int nd_send(netdev_t *netdev, const iolist_t *iolist)
 
     mutex_lock(&dev->devlock);
 
-#ifdef MODULE_NETSTATS_L2
-    netdev->stats.tx_bytes += count;
-#endif
+    if (cmd_rcr(dev, REG_ECON1, -1) & ECON1_TXRTS)
+    {
+        /* there is already a transmission in progress */
+        if (xtimer_now_usec() - dev->tx_time > 2429)
```

Comment:
please use macros or constants (with proper documentation) instead of magic number like `2429`

Reply:
The problem was, that I had no idea how to name this timeout since it is not defined as a parameter in the IEEE802.3 standard. Therefore, I simply used the value as defined in the errata sheet even though they defined this timeout for half duplex mode. In switched environments with full-duplex this time should be much smaller. The transmission of an Ethernet frame with maximum length of 1526 byte (including the 8 bytes preamble and start frame delimiter) should not take longer than 1.2208 ms at 10 Mbps. So, the magic time defined in the errata sheet seems to be the double of the maximum packet transmission time with some additional guard period between the transmissions.

I'm not sure whether RIOT is able at all to send a second packet within these 1.2208 ms. If not, we could remove the timeout guard completely. In that case, it would be enough to test the ```ECON1_TXRTS``` flag before next packet is sent and to reset the transmission logic, if the previous transmission has not been finished.